ui/src/app/store/device/slice.ts

```diff
@@ -0,0 +1,32 @@
+import { SliceCaseReducers } from "@reduxjs/toolkit";
+import { Machine } from "../machine/types";
+import { generateSlice, GenericSlice } from "../utils";
+import { Device, DeviceState } from "./types";
+
+type DeviceReducers = SliceCaseReducers<DeviceState>;
+
+export type DeviceSlice = GenericSlice<DeviceState, Device, DeviceReducers>;
+
+const DeviceSlice = generateSlice<
+  Device,
+  DeviceState["errors"],
+  DeviceReducers
+>({
+  name: "device",
+  reducers: {
+    createWithMachines: {
```

Comment:
I think this is a leftover from the resourcepool slice file. I'm pretty sure we don't need any extra reducers for devices.

Reply:
Good catch, thanks!